docs.json

```diff
@@ -212,6 +212,12 @@
                   "editor/share-links/framer-and-rive"
                 ]
               },
+              {
+                "group": "MCP",
```

Comment:
Does everyone know what this is? I didn't before rive.  Do we need to say mcp (ai) or something?

Reply:
My gut is telling me this is geared towards people who _know_ what MCP is, since it's a _bit_ of an involved process right now, and not just a site you'll visit for AI interactivity. But, if users are finding this confusing, we can always change it; I don't think updating the group would require a redirect.

Reply:
@RobertHaynie Has there been discussion on MCP vs something like MCP (AI)?